### PR TITLE
[RW-1579][Risk=no] removing nav bar from notebook view

### PR DESCRIPTION
**Pull request by UrsaStutsman**
@calbach Think I've addressed everything
@calbach sorry about that, got interrupted and didn't finish that thought. Should work now.
@calbach So I did some small tweaks, but I can't be positive if they fixed the issue for you as I am not seeing the same thing you are? If this still doesn't resolve, can you try tweaking on your machine until it looks right, and then I can pull those changes in here if they still look goon on mine?
@calbach so, yeah not sure, but apparently my bottom scroll bar disappears if I'm not using it? Or if it's not wide enough, so I wasn't seeing it, or the double scroll bar on the right without doing some specific moves. Regardless, I think I got everything this time. Let me know any thoughts.

### CODE REVIEW

File: ui/src/app/views/notebook-redirect/component.css [hunk 1]
```diff
@@ -77,5 +77,14 @@
 }
 
 .i-frame {
-  height: 0;
+  width: 104%;
```
[the remark this answers is not in the document]
Comment (UrsaStutsman) [reply ui/src/app/views/notebook-redirect/component.css]: When I did not have this, the i frame seemed to not quite reach the left edge of the screen.

File: ui/src/app/views/notebook-redirect/component.css [hunk 2]
```diff
@@ -77,5 +77,14 @@
...
+  margin-left: -.5rem;
```
[the remark this answers is not in the document]
Comment (UrsaStutsman) [reply ui/src/app/views/notebook-redirect/component.css]: I am not seeing any.

File: ui/src/app/views/notebook-redirect/component.html [hunk 3]
```diff
@@ -38,5 +38,5 @@ <h2 class="notebooks-header" *ngIf="creating">Creating New Notebook '{{notebookN
 </div>
 <div *ngIf="progress === Progress.Redirecting || progress === Progress.Loaded" [class.hidden]="progress === Progress.Redirecting" class="i-frame">
   <div id="seperator" *ngIf="progress === Progress.Loaded"></div>
-  <iframe id="leo-iframe" style="width:100%; height: 33rem;" frameBorder="0" [src]="leoUrl"></iframe>
+  <iframe id="leo-iframe" style="width:100%" frameBorder="0" [src]="leoUrl"></iframe>
```
Comment (calbach) [inline ui/src/app/views/notebook-redirect/component.html]: Does this work for you? I pulled your branch and it does not seem to be working.

image

**Comment by calbach** [comment]
I'm failing to see how this could be machine/browser dependent. I would expect that all browsers will have window scroll bars in the current PR. Lets make sure we're talking about the same problem before continuing.

CURRENT PR (scroll window down and right to match):
image

DESIRED:
image


Here is what worked for me locally:

a. Manually remove the outermost app padding, and revert to 100% width (no negative margins)
b. Set iframe height to `calc(100vh - 4rem)`

To implement (a) cleanly, we would want to have some kind of style toggle (similar to what you did in workspaceNavBar) in signed in component for that padding left/right. Possibly you could combine these route attributes by renaming to `reducedChrome: true` or `minimizeChrome: true`. This would replace the hacky negative margins.

For (b), ideally we'd fix the page flow so we could instead just use 100%. But we do have this hack elsewhere so it's probably fine to just go with that height: https://github.com/all-of-us/workbench/blob/5fb16d81d6d408b2aabd44b49e47271a9622ec12/ui/src/app/views/signed-in/component.css#L67

File: ui/src/app/views/notebook-redirect/component.css [hunk 4]
```diff
@@ -77,5 +77,17 @@
...
+  height: 100%;
+  margin-left: -.5rem;
+  margin-right: -.5rem;
+}
+
+.hidden {
+  display: none;
+}
+
+#seperator {
```
Comment (calbach) [inline ui/src/app/views/notebook-redirect/component.css]: separator

**Pull request merged**